src/driver/experimental_trial.py

```diff
@@ -125,9 +125,9 @@ def evaluate(self, rng: np.random.Generator) -> pd.DataFrame:
             halo, self._trial_descriptor.system_params, privacy_budget
         )
 
-        test_points = self._trial_descriptor.experiment_params.generate_test_points(
+        test_points = list(self._trial_descriptor.experiment_params.generate_test_points(
```

Comment:
https://github.com/world-federation-of-advertisers/planning-evaluation-framework/blob/26ef53422c3ab83b6336cd656643468770536998/src/driver/experiment_parameters.py#L67

If ```generate_test_points``` doesn't return List, the above typing description should be modified.

Reply:
Thanks for pointing this out.  I changed the signature to return Iterable[ReachPoint].